pydrive2/files.py

```diff
@@ -518,6 +518,30 @@ def Delete(self, param=None):
         """
         self._FilesDelete(param=param)
 
+    @LoadAuth
+    def Copy(self, target_folder: "GoogleDriveFile", new_title = None):
+        """Copy this file to a new location.
+
+        :param target_folder: Folder where the file will be copied.
+        :type target_folder: GoogleDriveFile.
+        :param new_title: Name of the new file.
+        :type new_title: str.
+        """
+
+        if new_title is None:
+            new_title = self["title"]
+
+        body = {
+            "parents": [{"id": target_folder['id']}],
+            'title': new_title 
+            }
+            
+        self.auth.service.files().copy(
```

Comment:
PTAL at the other methods around, this call should be wrapped with `try ... except errors.HttpError as error: ... raise ApiRequestError(error)`

Reply:
resolved in [748ee9b](https://github.com/iterative/PyDrive2/pull/188/commits/748ee9bb4d4b9178c8876b0f3ffd897c92e2c4b1)